tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -88,5 +90,539 @@ public void SubscribeToAppDomainUnhandledException ()
 			}, Path.Combine (Root, builder.ProjectDirectory, "startup-logcat.log"), 45), $"Output did not contain {expectedLogcatOutput}!");
 		}
 
+
+		void SymbolicateAndAssert (string symbolArchivePath, string logcatFilePath, IEnumerable<string> expectedStackTraceContents)
+		{
+			// 09-22 14:21:07.064 12786 12786 I MonoDroid:   at UnnamedProject.MainActivity.OnCreate (Android.OS.Bundle bundle) [0x00051] in <b3164619c4824e379aecfb7335bd4cce>:0
+			var obfuscatedStackRegex = new Regex ("in <.*>:0");
```

Comment:
You might make this a member variable to reuse it and use `RegexOptions.Compiled`:
```csharp
Regex obfuscatedStackRegex = new Regex ("in <.*>:0", RegexOptions.Compiled);
```